src/hmn_extras.c

```diff
@@ -88,7 +88,6 @@ int libxmp_hmn_new_instrument_extras(struct xmp_instrument *xxi)
 	if (xxi->extra == NULL)
 		return -1;
 	HMN_INSTRUMENT_EXTRAS((*xxi))->magic = HMN_EXTRAS_MAGIC;
-
```

Comment:
See previous comments.

Reply:
I see no issues with this: all other lines in these tiny procs have no newlines between them.

Reply:
kept this one.